docs/additional-features/chainbridge/setup.md

```diff
@@ -0,0 +1,193 @@
+---
+id: setup
+title: Setup
+---
+
+## Contracts deployment
+
+In this section, you will deploy the required contracts to the Polygon PoS and Polygon Edge chain with `cb-sol-cli`.
+
+```bash
+# Setup for cb-sol-cli command
+$ git clone https://github.com/ChainSafe/chainbridge-deploy.git
+$ cd chainbridge-deploy/cb-sol-cli
+$ make install
+```
+
+Firstly, we will deploy contracts to the Polygon PoS chain by `cb-sol-cli deploy` command.  `--all` flag makes the command deploy all the contracts, including Bridge, ERC20 Handler, ERC721 Handler, Generic Handler, ERC20, and ERC721 contract. In addition, it'll set the default relayer account address and the threshold
+
+```bash
+# Deploy all required contracts into Polygon PoS chain
+$ cb-sol-cli deploy --all --chainId 99 \
+  --url https://rpc-mumbai.matic.today \
+  --privateKey [ADMIN_ACCOUNT_PRIVATE_KEY] \
+  --relayers [RELAYER_ACCOUNT_ADDRESS] \
+  --relayerThreshold 1
+```
+
+
+Learn about chainID and JSON-RPC URL [here](https://edge-docs.polygon.technology/docs/additional-features/chainbridge/definitions)
```

Comment:
Please don't attach protocol and domain for internal link. This link will be broken when changing domain. You can just set the path for internal link

```suggestion
Learn about chainID and JSON-RPC URL [here](/docs/additional-features/chainbridge/definitions)
```